feat(content) Remnant: Celebration of Remembrance with Taely (#7070)
endless-sky/endless-sky: data/remnant/remnant 2 side missions.txt

			label mainreturn
			`	Taely leads you around the area for a few more hours, pointing out historical features, places where they resurrected trades and skills from before the advent of flight, and other interesting things. Eventually she heads back to the train station, which deposits you back in the main spaceport.`
			`	"I hope you enjoyed the tour, <first>. It was interesting to see our history through fresh eyes." She smiles micheviously. "And I enjoyed the hoverboarding, too. Maybe we can do it again sometime." She pauses for a moment, focusing on something you cannot see. "Looks like duty calls. Embers guide your way, Captain <last>." She waves, and heads off into the crowd.`

mission "Remnant: Celebration 1"
	name "Deliver Celebration records to <destination>"

> 	description "The Remnant have asked you to take records of their celebration to <destination>"

Reviewer: ```suggestion
	description "The Remnant have asked you to take records of their celebration to <destination>."
```

	minor
	source
		government "Remnant"
	destination "Ssil Vida"
	to offer
		has "event: remnant: cognizance calm"
		has "license: Remnant Capital"

> 		# random < 50

Reviewer 2: So no one forget about this.
```suggestion
		random < 50
```

		"day" = 7
	on offer
		conversation

> 			`You stop by the starport just in time to see a large number of Remnant going somewhere together. A few of them notice you and gesture an invitation to accompany them, but before you can react Taely appears from the stream of people and approaches you.`

Author: This is intended to be fairly fast sequence. I don't personally feel that I am frozen in place while Taely shows up, but I can see about tweaking that.

			`	"Well met, <first>. We were just going to the monthly Celebration of Remembrance. Would you like to accompany us?"`
			choice
				`	"Yes, I would."`
					goto celebrationattendance
				`	"What is that?"`
					goto celebrationdefinition
				`	"No thanks."`
			label celebrationdeclined
			`	She nods. "Well, may the Embers guide you, <first>." With that, she rejoins the stream of people as they vanish from sight.`
				decline
			label celebrationdefinition
			`	"The Celebration of Remembrance is a gathering; we take a moment to remember where we have come from, appreciate where we are, and express hope for where we will be in the future." Her gestures take on a critical tone. "That sounds very formal, but it is more artistic than defined." She glances at the slowing flow of people. "It will be starting shortly, and I want to attend this month, so I should get moving. Would you like to join us?"`
			choice

> 				`	"Yes, I do."`

Reviewer 2: "Yes, I do [like to join you.]"
That doesn't sound right.
```suggestion
				`	"Yes, I would."`
```

				`	"No thanks."`
					goto celebrationdeclined
			label celebrationattendance

> 			`	As the two of you join the flow of people Taely gestures once again. "If you would like, I could provide you with some commentary and explanation, but it is more about feeling than doing anything specific. Together, we remember and celebrate through free-form dance mixed with singing and music. You are welcome to participate, but try not to disrupt or impede others in the way you join the dancing."`

Reviewer: ```suggestion
			`	As the two of you join the flow of people, Taely gestures once again. "If you would like, I could provide you with some commentary and explanation, but it is more about feeling than doing anything specific. Together, we remember and celebrate through free-form dance mixed with singing and music. You are welcome to participate, but try not to disrupt or impede others in the way you join the dancing."`
```

			choice
				`	"No, that is alright. I will just experience it."`
				`	"Yes, that would be appreciated."`
					goto celebrationannotated
			`	The flow of people you are a part of joins with other streams of people, finally leading out of the typical hallways and into a large cave. A large opening on one side opens upwards towards the night sky, framed by rough stone and a handful of shield emitters. Stalagmites ring the perimeter of the room with matching stalactites above them, but there is no trace of drips or moisture here. While the walls are rough natural stone, the floor and stalagmites have been polished smooth. At a quick glance, you estimate that there are probably over a hundred people in here with more filtering in through several doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`

> 			`	Without warning the room stills, and the doors close. You see a flurry of gestures wash over the room, as if everyone is repeating what someone closer to the center is saying, but you cannot see anyone starting it. "Perimeter secure?" it asks, followed by a ripple of gestures from the edges of the room back toward the center: "All clear."`

Author: Nope, this is deliberate. The protagonist is not aware of any noise or obvious reason for the room to still. It is intended to be incongruous and something that the player might take note of.

> 			`	Suddenly you are aware of a drum beat building in the silence, and realize that it had started with the first ripple of gestures, so quiet that it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`

Reviewer 2: ```suggestion
			`	Suddenly, you are aware of a drum beat building in the silence, and realize that it had started with the first ripple of gestures, so quiet that it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
```

			`	"Thump."`
			`	"Thump."`
			`	"Thump."`
			`	"Thump."`
			choice
				`	(Look around the room.)`

> 				`	(Move in time with the beat.)`

Author: Correct. This is both a break in the flow and an opportunity to let the player do something. Neither option affects the story, though. Which is logical, since the player isn't the focus of what is going on, and has no bearing on the events. As such, this is more an RP detail so that players can decide for themselves if they're just observing or actually taking part.

> 			`	The beat echoes in the cavern, but you quickly manage to locate the source: An elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`

Reviewer 3: ```suggestion
			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
```

> 			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`

Reviewer 2: Are hollow stalagmites a thing in real life?
It seems hollow stalactites are a thing, but not so much stalagmites?
Is the implication that the dripping that usually forms these happened upwards, for some reason, I guess someone was screwing with gravity?
Or, something else? Are they just a different type to what we might have on Earth? Are they supposed to be artificial?

Author: Go with the simplest explanation: They're artificially hollowed out. I'm not actually averse to the other options (such as someone messing with gravity or that they are somehow naturally like that), but simply that they've been hollowed out artificially somehow is the main thing.

			`	"Thump-thump."`
			`	"Thump-thump."`
			`	"Thump-thump."`

> 			`	"Thump-thump."`

Author: It is intended to remind a person of a heartbeat.

> 			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and others have joined hands and move around in a circle. Everywhere you look there are people dancing in countless variations.`

Reviewer 3: ```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and have begun to move around in a circle. Everywhere you look there are people dancing in countless variations.`
```

> 			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and have begun to move around in a circle. Everywhere you look there are people dancing in countless variations.`

Author: The circle as a whole is moving in place, as in the circle doesn't change location, just the people who are a part of it moving around the circle.

> 			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and have begun to move around in a circle. Everywhere you look there are people dancing in countless variations.`

Reviewer 2: Then, I would expect "moving in a circle" or perhaps "moving around a circle".

> 			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving around in a circle. Everywhere you look there are people dancing in countless variations.`

Reviewer 2: ```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving around in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving around in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`
```

			`	Out of nowhere you here a voice singing. You cannot make out what it is saying, but the tone sounds like one of anticipation. You can, however, clearly make out a chant that cuts through the solo voice in time with the drumming:`
			`	"...6...5...4...Ignition...2...1...Liftoff!"`
			choice
				`	(Look around with anticipation.)`
					goto countdown
				`	(Look around nervously.)`
			`	Taely notices your anxiety and gestures reassuringly. "All is well."`
			label countdown
			`	On the final word the chant ends and dozens of deep voices join in with a roaring crescendo that builds then fades away as many new voices join in with tones of wonder. At this point, it seems that almost everyone in the room is either drumming, dancing, or singing - and many seem to be doing several simultaneously.`

> 			`	As the stars wheel outside the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion of the singing is clear. The early tone of anticipation shifted to the roar of excitement then on to a peaceful tone of wonder. As the ceremony continues, this shifts to a steady, almost mechanical feel, which is then penetrated by a piercing thread of excitement that leads to the entire tone of the song becoming one of curiosity with a driving beat. The melody wanders through regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other emotions before finally culminating in a return to the original heartbeat as the singers' voices reach upwards in a tone of hope and then everything trails off into silence.`

Author: It is intended to evoke the passage of time. This part of the description steps  back from an actual description of the sequence of events to a larger "big picture" overview of the event.

			choice

> 				`	Stand still and absorb the moment.`

Reviewer 3: ```suggestion
				`	(Stand still and absorb the moment.)`
```

> 				`	Look around to see what is happening next.`

Reviewer 3: ```suggestion
				`	(Look around to see what is happening next.)`
```

> 			`	Around you, everyone stands still. Most look exhausted, some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the cave is nearly empty.`

Reviewer 3: ```suggestion
			`	Around you, everyone stands still. Most look exhausted, and some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the cave is nearly empty.`
```

			`	Taely gestures quietly to you. "It will start with the ritual query and response asking if we are safe. Safe in this moment, at least."`
			`	Without warning the room stills, and the doors close. You see a flurry of gestures wash over the room, as if everyone is repeating what someone closer to the center is saying, but you cannot see anyone starting it. "Perimeter secure?" it asks, followed by a ripple of gestures from the edges of the room back toward the center: "All clear."`
			`	Suddenly you are aware of a drum beat building in the silence, and realize that it had started with the first ripple of gestures, so quiet that it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
			`	"Thump."`
			`	"Thump."`
			`	"Thump."`

> 			`	"Thump."`

Reviewer: I'm cool with the thumping, but I don't quite like the progression idea. There are only a few variations we can do (in fact you listed them: `thump, Thump, THUMP`). Unless there are literally exactly three "thumps" happening, I think the starkness of the word change takes away from the reader's personal ability to interpret "gradualness", if that makes any sense.
Basically, each word change (`thump -> Thump`) would feel more like a leap rather than a slower, more gradual increase to me.

> 			`	"Thump."`

Author: Given W1zrad's comments, I prefer to leave this as is. While there is progression here; it is far more than just three thumps. This is more of a crescendo over probably at least a dozen thumps.

> 			`	Suddenly you become aware of a drum beat building in the silence, and then realize that it had already started with the first ripple of gestures, so quiet that, until now, it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`

Reviewer 2: ```suggestion
			`	Suddenly, you become aware of a drum beat building in the silence, and then realize that it had already started with the first ripple of gestures, so quiet that, until now, it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
```

			choice
				`	(Look around the room.)`
				`	(Move in time with the beat.)`
			`	At your side, Taely gestures again. "Existence starts with the patterns of the universe, and our story starts with life."`

> 			`	The beat echoes in the cavern, but you quickly manage to locate the source: An elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`

Reviewer 3: ```suggestion
			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to thump on a polished stalagmite. Judging by the resonance, it appears that it must be hollow. As you watch, a child steps up next to her and starts hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
```

> 			`	"Thump-thump."`

Author: I don't have time to type out the addition part right now, but will tomorrow about this time. So I'm leaving the suggestions that need me to do additional input un-committed so as to remind myself to do them tomorrow.

> 			`	"Thump-thump."`

Reviewer: See above - I personally prefer it as-is.

> 			`	"Thump-thump."`

Author: Likewise, I think I prefer to leave this as-is, as per W1zrad's comments.

			`	"Thump-thump."`
			`	"Thump-thump."`
			`	"Thump-thump."`

> 			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and others have joined hands and move around in a circle. Everywhere you look there are people dancing in countless variations.`

Reviewer 3: ```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you here other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and have begun to move around in a circle. Everywhere you look there are people dancing in countless variations.`
```

> 			`	Looking around the room you can see people starting to move in time with the beat. Some are already swaying their whole body while others are just nodding their heads or tapping their feet. Standing next to you, Taely has a hand over her heart, tapping softly in time. Soon you hear other drummers join in and the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun to move around in a circle. Soon, everywhere you look there are people dancing, alone and together, in countless variations.`

Reviewer 2: ```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving around in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates through the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`
```
```suggestion
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`
```

			`	As the stars wheel outside the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion of the singing is clear. The early tone of anticipation shifted to the roar of excitement then on to a peaceful tone of wonder. As the ceremony continues, this shifts to a steady, almost mechanical feel, which is then penetrated by a piercing thread of excitement that leads to the entire tone of the song becoming one of curiosity with a driving beat. The melody wanders through regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other emotions before finally culminating in a return to the original heartbeat as the singers' voices reach upwards in a tone of hope and then everything trails off into silence.`

> 			`	Through it all, Taely offers a running commentary on how each stretch of the ceremony represents a different piece of their collective history, ranging from the launch of the first human into space to its commercialization, and then the new age of exploration opened by the advent of hyperdrives. It also includes numerous conflicts, disasters, and wars. Much of it is familiar to you from what you know of human history, but the later part is strange and unfamiliar. Filled with stories of first contact with the aliens, discovery of strange ruins, alien invasions, desperate last stands, and brilliant discoveries, you are left with the impression that despite the Remnant's current security, they have been on the precipice of extinction for much of their existence.`

Author: There are now fewer discoveries. lol.

> 			`	Through it all, Taely offers a running commentary on how each stretch of the ceremony represents a different piece of their collective history, ranging from the launch of the first human into space to its commercialization, and then the new age of exploration opened by the advent of hyperdrives. It also includes numerous conflicts, disasters, and wars. Much of it is familiar to you from what you know of human history, but the later part is strange and unfamiliar. Filled with stories of first contact with the aliens, discovery of strange ruins, alien invasions, desperate last stands, and brilliant discoveries, you are left with the impression that despite the Remnant's current security, they have been on the precipice of extinction for much of their existence.`

Author: Likewise, fewer discoveries.

			choice

> 				`	Stand still and absorb the moment.`

Reviewer 3: ```suggestion
				`	(Stand still and absorb the moment.)`
```

> 				`	Look around to see what is happening next.`

Reviewer 3: ```suggestion
				`	(Look around to see what is happening next.)`
```

> 			`	Around you, everyone stands still. Most look exhausted, some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the cave is nearly empty.`

Reviewer 3: ```suggestion
			`	Around you, everyone stands still. Most look exhausted, and some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the cave is nearly empty.`
```

			label celebrationend
			`	"I always find these Celebrations of Remembrance to be invigorating. What did you think of it, <first>?"`
			choice
				`	"I think I will have to get back to you on that later."`
					goto celebrationrequest
				`	"That was awe-inspiring."`
					goto awestruck
				`	"Why were some of the people crying?`
					goto askcrying
				`	"Who leads these ceremonies?"`
					goto ceremonyleader

> 				`	"That seemed both well rehearsed and very disorganized."`

Reviewer 3: ```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

> 			`	"I always find these Celebrations of Remembrance to be invigorating. What did you think of it, <first>?"`

Reviewer: ```suggestion
			`	"I always find these Celebrations of Remembrance to be invigorating," Taely says. "What did you think of it, <first>?"`
```

			label celebrationrehearsed

> 			`	Taely smiles at that. "Well, it is and it is not. We hold these several times a month, with the expectation that people attend no more than one of them. So most of us have attended hundreds of these. Although as far as that goes, every one of these celebrations is different. Each person's participation is up to them, and so the details and feel of each celebration are unique every time." She makes a gesture you have come to realize amounts to "If you have more questions, ask them."`

Author: That is meant to be no more than one per month, so a dozen or so per year. Works out to about 120/decade. Over the course of the average human life could easily surpass a thousand.

> 			`	Taely smiles at that. "Well, it is and it is not. We hold these several times a month, with the expectation that people attend no more than one of them. So most of us have attended hundreds of these. Although as far as that goes, every one of these celebrations is different. Each person's participation is up to them, and so the details and feel of each celebration are unique every time." She makes a gesture you have come to realize amounts to "If you have more questions, ask them."`

Author: Alright, this has been edited to: 

> "Well, it is and it is not. We hold these several times a month, with most people taking part in one every few weeks. So most of us have attended hundreds of these. 

The point was to convey three ideas here:

1. These happen on a regular basis, several times a month.
2. Most people attend these, but not every one of them.
3. These are not formal "everyone is required to attend this" obligations.

> 			`	Taely smiles at that. "Well, it is and it is not. We hold these several times a month, with most people taking part in one every few weeks. So most of us have attended hundreds of these. Although as far as that goes, every one of these celebrations is different. Each person's participation is up to them, and so the details and feel of each celebration are unique every time." She makes a gesture you have come to realize amounts to "If you have more questions, ask them."`

Reviewer 2: What she said earlier made it seem like this was a monthly thing.

Author: It is, sort of. The event itself takes place many times per month, but it is generally considered to be something that a person only attends on roughly monthly basis. Enough of these take place in a month that all the Remnant present on that world would be able to attend. There's probably quite a few going on any particular day.

This is sort of like Friday religious services (or Saturday, or Sunday, depending on one's religion of choice); except that the cultural expectation is of monthly attendance, not weekly.

On a more subtle level, this is intended to show that yes, there is religion here; but it's much more casual and informal than many real-life present-day religious denominations.

			choice
				`	"I think that is all for now."`
					goto celebrationrequest
				`	"That was awe-inspiring."`
					goto awestruck
				`	"Why were some of the people crying?`
					goto askcrying
				`	"Who leads these ceremonies?"`
			label ceremonyleader
			`	Taely gestures take on a tone of conviction. "For something like this, or anything of a social or spiritual nature for that matter, we determine who takes the lead for various parts at the moment based on who is present. Whether we are talking about our connection with society, or our connection to the Universe, we must each forge our own understanding of it. As such, when we gather together for ceremonies such as this, virtually anyone could be chosen to take the lead for one of the segments.`

> 			`	"As a result, each celebration takes on a different feel - after all, each of us has our own unique perspective, and bring our own experiences to contribute." She pauses, and adopts a respectful tone. "The ceremony is not just about major events, either. Everyone who takes part brings their own life experiences and memories to it. Memories of friends and family who are no longer with us are common."`

Author: > Memories of friends and family who are no longer with us are a common as well.

That "a" near the end doesn't seem right.

> 			`	"As a result, each celebration takes on a different feel - after all, each of us has our own unique perspective, and bring our own experiences to contribute." She pauses, and adopts a respectful tone. "The ceremony is not just about major events, either. Everyone who takes part brings their own life experiences and memories to it. Memories of friends and family who are no longer with us are common."`

Reviewer 4: I think it was originally for something like "a common sight" but decided against.

Fixed :)

			choice
				`	"I think that is all for now."`
					goto celebrationrequest
				`	"That was awe-inspiring."`
					goto awestruck
				`	"Why were some of the people crying?`
					goto askcrying

> 				`	"That seemed both well rehearsed and very disorganized."`

Reviewer 3: ```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

					goto celebrationrehearsed
				`	"You said 'virtually.' Who is omitted?"`

> 			`	Taely scowls a bit at this. "Only those of guilty of the most serious crimes cannot stand as a lead." She pauses, and continues with an amused tone. "Even children take the lead now and then, usually with a parent or sibling to guide them." She smiles at that, then gestures for you to continue asking your questions.`

Reviewer 2: ```suggestion
			`	Taely scowls a bit at this. "Only those guilty of the most serious crimes cannot stand as a lead." She pauses, and continues with an amused tone. "Even children take the lead now and then, usually with a parent or sibling to guide them." She smiles at that, then gestures for you to continue asking your questions.`
```

			choice
				`	"I think that is all for now."`
					goto celebrationrequest
				`	"That was awe-inspiring."`
					goto awestruck
				`	"Why were some of the people crying?`
					goto askcrying

> 				`	"That seemed both well rehearsed and very disorganized."`

Reviewer 3: ```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

					goto celebrationrehearsed
				`	"What constitutes your most serious crimes?"`

> 			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective wellbeing is the single most destructive and senseless acts someone can do."`

Reviewer 3: ```suggestion
			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective wellbeing is the single most destructive and senseless act someone can do."`
```

			choice
				`	"I think that is all for now."`
					goto celebrationrequest
				`	"That was awe-inspiring."`
					goto awestruck
				`	"Why were some of the people crying?`

> 				`	"That seemed both well rehearsed and very disorganized."`

Reviewer 3: ```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

					goto celebrationrehearsed
			label askcrying

> 			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`

Reviewer 3: ```suggestion
			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`
```

> 			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`

Reviewer 4: ```suggestion
			`	Taely gestures gently. "Many were friends of the crew of a ship that was recently destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals. Sometimes the whole experience is simply overwhelming, and some of us still sorrow for other losses in our stories."`
```

> 			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`

Reviewer 2: Is this the *only* reason? This whole seemed like a pretty emotional experience. I'd imagine some people would cry of joy as well, and both crying of joy and sorrow could be for a variety of reasons.

Author: There is, in all likelyhood, a variety of reasons. It is a fairly emotional thing. I didn't really want to expound at length on all the many different ways there might be someone crying.

			choice
				`	"I think that is all for now."`
					goto celebrationrequest
				`	"That was awe-inspiring."`
				`	"Who leads these ceremonies?"`
					goto ceremonyleader

> 				`	"That seemed both well rehearsed and very disorganized."`

Reviewer 3: ```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

					goto celebrationrehearsed
			label awestruck
			`	Taely nods in agreement and simply gestures expansively at the cavern around you, giving you a few moments to contemplate it. Eventually she gestures asking if you have any further questions.`
			choice
				`	"I think that is all for now."`
					goto celebrationrequest
				`	"Why were some of the people crying?`
					goto askcrying
				`	"Who leads these ceremonies?"`
					goto ceremonyleader

> 				`	"That seemed both well rehearsed and very disorganized."`

Reviewer 3: ```suggestion
				`	"That seemed both well-rehearsed and very disorganized."`
```

					goto celebrationrehearsed
			label celebrationrequest

> 			`	Taely strides over to a console near the door and retrieves a crystal from a compartment within the console. She slips it into a case and turns to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present can experience it too. This recording is destined for those working in Postverta, as a friend of one of those being mourned today is stationed there. Could you deliver this to them please?"`

Reviewer 4: ```suggestion
			`	Taely strides over to a console near the closest door and retrieves a crystal from a compartment within. She slips it into a case and turns back to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present at a particular event may still experience it too. This recording is destined for those working in Postverta, as a friend of one of those being mourned today is stationed there. Could you deliver this to them please?"`
```

> 			`	Taely strides over to a console near the door and retrieves a crystal from a compartment within the console. She slips it into a case and turns to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present can experience it too. This recording is destined for those working on Postverta, as a friend of one of those being mourned today is stationed there. Could you deliver this to them please?"`

Reviewer 2: ```suggestion
			`	Taely strides over to a console near the door and retrieves a crystal from a compartment within the console. She slips it into a case and turns to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present can experience it too. This recording is destined for those working in Postverta, as a friend of one of those being mourned today is stationed there. Could you deliver this to them please?"`
```

> 			action set "Remnant: Attended Celebration of Remembrance with Taely"

Reviewer 2: Does this work?

Author: It did not. Now it does. Thanks for asking!

			choice
				`	"Yes."`
					accept

> 				`	"No."`

Reviewer 3: The option to reject a mission is usually less blunt than this.
```suggestion
				`	"Sorry, I'm not headed that way."`
```

			`	She looks disappointed, but merely shrugs and pockets the crystal. "I will find someone else to transport this, then."`
				decline
	on complete
		payment 50000
		conversation

> 			`When you arrive on Postverta there is a small group of Remnant waiting at the landing bay for you. As they accept the crystal from you one gestures "Thank you for bringing this to me, Captain <first>. Being able to join in the Remembrance will help me to remember and celebrate those who have gone on before." They head off into the depths of the station, leaving you in the bay.`

Reviewer 3: ```suggestion
			`When you arrive on Postverta there is a small group of Remnant waiting at the landing bay for you. As they accept the crystal from you one gestures, "Thank you for bringing this to me, Captain <first>. Being able to join in the Remembrance will help me to remember and celebrate those who have gone on before." They head off into the depths of the station, leaving you in the bay.`
```

> 			`When you arrive on Postverta there is a small group of Remnant waiting at the landing bay for you. As they accept the crystal from you one gestures, "Thank you for bringing this to me, Captain <first>. Being able to join in the Remembrance will help me to remember and celebrate those who have gone on before." They head off into the depths of the station, leaving you in the bay.`

Reviewer 4: ```suggestion
			`When you arrive on Postverta there is a small group of Remnant waiting at the landing bay for you. As you hand the crystal to one, another gestures, "Thank you for bringing this to us, Captain <first>. Being able to join in the Remembrance will help me to celebrate those who have gone on before." The group heads off into the depths of the station, leaving you in the bay.`
```

> 			`	Taely never mentioned anything about compensation, but as soon as the Remnant left your commlink notifies you that you have been assigned <payment>.`

Reviewer 2: ```suggestion
			`	Taely never mentioned anything about compensation, but as soon as the Remnant leave, your commlink notifies you that you have been assigned <payment>.`
```
```suggestion
			`	Taely never mentioned anything about compensation, but as soon as the Remnant have left, your commlink notifies you that you have been assigned <payment>.`
```

mission "Remnant: Celebration 1"
	name "Deliver Celebration records to <destination>"
	description "The Remnant have asked you to take records of their celebration to <destination>."
	minor
	source
		government "Remnant"
	destination "Ssil Vida"
	to offer
		has "event: remnant: cognizance calm"
		has "license: Remnant Capital"
		random < 50
		"day" = 7
	on offer
		conversation
			`You stop by the starport just in time to see a large group of Remnant departing. A few of the nearest notice your interest and begin gesturing recognition or various invitations to accompany them, but before you are able to respond, Taely appears from the stream of people and approaches you.`
			`	"Well met, <first>. We were just going to the monthly Celebration of Remembrance. Would you like to accompany us?"`
			choice
				`	"Yes, I would."`
					goto celebrationattendance
				`	"What is that?"`
					goto celebrationdefinition
				`	"No thanks."`
			label celebrationdeclined

> 			`	She nods. "Well, may the Embers guide you, <first>." With that, she rejoins the stream of people as they vanish from sight.`

Author: Hmm, that makes it sound more like Taely vanishing, as opposed to the stream.

> 			`	She nods. "Well, may the Embers guide you, <first>." With that, she rejoins the stream of people as they vanish from sight.`

Reviewer 4: It would, but that's a change I made when I switched the wording. Thought it would help address the awkwardness @quyykk mentioned to talk about when you could no longer see the group versus when you could no longer see her.

But suggestion from @TheMarksman-ES is improvement on original even though switching that back.

> 			`	She nods. "Well, may the Embers guide you, <first>." With that, she rejoins the stream of people as they vanish from sight.`

Author: Would it be acceptable to just leave it as-is, then?

> 			`	She nods. "Well, may the Embers guide you, <first>." With that, she rejoins the stream of people as they vanish from sight.`

Author: Alright then, leaving it as-is. Thanks!

				decline
			label celebrationdefinition
			`	"The Celebration of Remembrance is a gathering; we take a moment to remember where we have come from, appreciate where we are, and express hope for where we will be in the future." Her gestures take on a critical tone. "That sounds very formal, but it is more artistic than defined." She glances at the slowing flow of people. "It will be starting shortly, and I want to attend this month, so I should get moving. Would you like to join us?"`
			choice
				`	"Yes, I would."`
				`	"No thanks."`
					goto celebrationdeclined
			label celebrationattendance
			`	As the two of you join the flow of people, Taely gestures once again. "If you would like, I could provide you with some commentary and explanation, but it is more about feeling than doing anything specific. Together, we remember and celebrate through free-form dance mixed with singing and music. You are welcome to participate, but try not to disrupt or impede others in the way you join the dancing."`
			choice
				`	"No, that is alright. I will just experience it."`
				`	"Yes, that would be appreciated."`
					goto celebrationannotated

> 			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually coming out of the typical hallways and into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or trace of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`

Reviewer 5: ```suggestion
			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually emerging from the typical hallways into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or traces of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
```

> 			`	Without warning the room stills, and all the doors close. A flurry of gestures washes out from the center of the room, as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`

Reviewer 5: ```suggestion
			`	Without warning, the room stills, and all the doors close. A flurry of gestures washes out from the center of the room as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`
```

Author: Are you sure? This doesn't feel like a good flow with the comma there.

Reviewer 6: I think both versions are correct in terms of grammar, so it is a question of which rhythm you prefer.

Reviewer 2: "Without warning the room stills, and all the doors close."
"Warning the room stills," sounds like giving a warning to "the room stills", whatever a "room still" is.
The 'without' means no such warning was given.
There probably should be a comma there.

Reviewer 6: > giving a warning to "the room stills"
I'm pretty sure [still can also be a verb](https://www.oxfordlearnersdictionaries.com/definition/english/still_4), in which case the comma isn't necessary.

Reviewer 4: I think it could go either way still, but @warp-core convinced me here it works best with the comma. :)

Author: I don't follow warp-core's example. The lack of warning is in regards to the room stilling. 

To me, having a comma there breaks the flow. "Without warning." Well, without warning about what?

It's not two different ideas, it's not even two different things influencing each other. It's one thing.

Reviewer 4: Without is a modifier to the phrase, not a part of it; the comma breaks up the phrase it's modifying.

"(((Without (warning the room stills)), (and all the doors close.))"

"(((Without (warning)), (the room stills), (and all the doors close.))"

Author: OK, well, at this point I kind of feel it's coming down to what is required by grammatical rules.

1. If a comma is grammatically *required* here, then I put it in.
2. If a comma is *not* required, then I would prefer to not have the comma as I feel it breaks the flow and the feel of what this detail is supposed to convey.

Reviewer 2: I believe the comma is grammatically required.

Author: This does not feel conclusive, but the members of the writing team who have commented seem to accept without a comma, so I am leaving it out.

			`	"Thump-thump."`
			`	"Thump-thump."`
			`	"Thump-thump."`
			`	"Thump-thump."`
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, some nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`
			`	Out of the chaos you hear a voice rise in song. You cannot make out what it is saying, but it carries the tones of anticipation. You can, however, clearly decipher a chant from many other voices, cutting through the solo voice in time with the drumming but not overwhelming either:`
			`	"...6...5...4...Ignition...2...1...Liftoff!"`
			choice
				`	(Look around with anticipation.)`
					goto countdown
				`	(Look around nervously.)`

> 			`	Taely notices your anxiety and gestures reassuringly. "All is well."`

Reviewer 5: ```suggestion
			`	Taely notices your anxiety and gestures reassuringly: "All is well."`
```

			label countdown
			`	On the final word the chant ends and dozens of deep voices join together in a roaring crescendo before fading away, replaced by many new voices expressing tones of wonder. At this point, it seems that almost every single person in the room is drumming, dancing, or singing - many seem to be doing several simultaneously.`

> 			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`

Reviewer 5: ```suggestion
			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`
```

> 			`	As the ceremony continues, this shifts again, to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other deeply emotional pairings before finally culminating in a return to the original, unified heartbeat as the singers' voices come together, reaching upwards in hope before finally everything trails off into silence.`

Reviewer 5: ```suggestion
			`	As the ceremony continues, this shifts again to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other deeply emotional pairings before finally culminating in a return to the original, unified heartbeat as the singers' voices come together, reaching upwards in hope before finally everything trails off into silence.`
```

			choice
				`	(Stand still and absorb the moment.)`
				`	(Look around to see what is happening next.)`
			`	Around you, everyone has gone still. Most look exhausted, and some are sitting on the floor or leaning against the walls. A few people look like they have been crying, and those nearby are hugging them or simply touching their shoulder. Most of the people you can see have reverent and joyful expressions. After the silence lingers for what feels like several minutes, the doors open and people start to leave. In short order the cave is nearly empty.`
				goto celebrationend
			label celebrationannotated

> 			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually coming out of the typical hallways and into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or trace of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`

Reviewer 5: ```suggestion
			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually emerging from the typical hallways into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or traces of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
```

			`	Taely gestures quietly to you. "We start with the ritual query and response asking if we are safe. Safe in this moment, at least."`

> 			`	Without warning the room stills, and all the doors close. A flurry of gestures washes out from the center of the room, as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`

Reviewer 5: ```suggestion
			`	Without warning, the room stills, and all the doors close. A flurry of gestures washes out from the center of the room as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`
```

Author: As per the other instance, I think 'Without warning the room stills,' sounds better as one phrase rather than two.

			`	Out of the chaos you hear a voice rise in song. You cannot make out what it is saying, but it carries the tones of anticipation. You can, however, clearly decipher a chant from many other voices, cutting through the solo voice in time with the drumming but not overwhelming either:`
			`	"...6...5...4...Ignition...2...1...Liftoff!"`
			choice
				`	(Look around with anticipation.)`
					goto countdownannotated
				`	(Look around nervously.)`
			`	Taely notices your anxiety and gestures reassuringly. "They sing of the days of humanity's infancy; when we first ventured out of the cradle."`
			label countdownannotated
			`	On the final word the chant ends and dozens of deep voices join together in a roaring crescendo before fading away, replaced by many new voices expressing tones of wonder. At this point, it seems that almost every single person in the room is drumming, dancing, or singing - many seem to be doing several simultaneously.`

> 			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`

Reviewer 5: ```suggestion
			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`
```

> 			`	As the ceremony continues, this shifts again, to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other deeply emotional pairings before finally culminating in a return to the original, unified heartbeat as the singers' voices come together, reaching upwards in hope before finally everything trails off into silence.`

Reviewer 5: ```suggestion
			`	As the ceremony continues, this shifts again to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other deeply emotional pairings before finally culminating in a return to the original, unified heartbeat as the singers' voices come together, reaching upwards in hope before finally everything trails off into silence.`
```

			choice
				`	(Stand still and absorb the moment.)`
				`	(Look around to see what is happening next.)`
			`	Around you, everyone has gone still. Most look exhausted, and some are sitting on the floor or leaning against the walls. A few people look like they have been crying, and those nearby are hugging them or simply touching their shoulder. Most of the people you can see have reverent and joyful expressions. After the silence lingers for what feels like several minutes, the doors open and people start to leave. In short order the cave is nearly empty.`
			label celebrationend

> 			`	"I always find these Celebrations of Remembrance to be invigorating," Taely says. "What did you think of it, <first>?"`

Reviewer 3: ```suggestion
			`	"I always find these Celebrations of Remembrance ceremonies to be invigorating," Taely says. "What did you think of it, <first>?"`
```

Reviewer 4: If adding "ceremonies," should become "Celebration of Remembrance" instead of "Celebrations."

Author: I get that it is technically correct; however I think it also makes sense that Taely just refers to it as "Celebrations of Remembrance" and omits the ceremonies label/descriptor.

Reviewer 3: Yeah looking back I don't know why I made this comment. Ignore it.

			choice
				`	"I think that is all for now."`
					goto celebrationrequest
				`	"That was awe-inspiring."`
					goto awestruck
				`	"Why were some of the people crying?`
					goto askcrying
				`	"That seemed both well-rehearsed and very disorganized."`
					goto celebrationrehearsed
				`	"You said 'virtually.' Who is omitted?"`
			`	Taely scowls a bit at this. "Only those guilty of the most serious crimes cannot stand as a lead." She pauses, and continues with an amused tone. "Even children take the lead now and then, usually with a parent or sibling to guide them." She smiles at that, then gestures for you to continue asking your questions.`
			choice
				`	"I think that is all for now."`
					goto celebrationrequest
				`	"That was awe-inspiring."`
					goto awestruck
				`	"Why were some of the people crying?`
					goto askcrying
				`	"That seemed both well-rehearsed and very disorganized."`
					goto celebrationrehearsed
				`	"What constitutes your most serious crimes?"`

> 			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective wellbeing is the single most destructive and senseless act someone can perpetrate."`

Reviewer 6: ```suggestion
			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective well-being is the single most destructive and senseless act someone can perpetrate."`
```

				`	"No, that is alright. I will just experience it."`
				`	"Yes, that would be appreciated."`
					goto celebrationannotated

> 			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually emerging from the typical hallways into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or traces of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`

Reviewer: ```suggestion
			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually emerging from the typical hallways into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up toward the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or traces of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
```

			`	"Thump."`
			`	"Thump."`
			`	"Thump."`
			`	"Thump."`
			choice
				`	(Look around the room.)`
				`	(Move in time with the beat.)`
			`	The beat echoes in the cavern, but you quickly manage to locate the source: an elderly Remnant using her hands to drum against a polished stalagmite that, judging by the resonance, appears to be hollow. As you watch, a child steps up next to her and starts also hitting the rock with their palms, slightly off-beat, changing the pattern into a hauntingly familiar one:`
			`	"Thump-thump."`
			`	"Thump-thump."`
			`	"Thump-thump."`
			`	"Thump-thump."`
			`	Looking around the room you can see people starting to move in time with the beat. Some are swaying, and some are nodding their heads or tapping their feet. Standing next to you, Taely has a hand on her heart, tapping softly. Soon you hear other drummers join in as the beat resonates throughout the room. Now people are not merely swaying to the beat, but dancing. Some spin in place, others rock back and forth between different stances, and a few have joined hands and begun moving in a circle. Everywhere you look there are people dancing in countless variations.`

> 			`	Out of the chaos you hear a voice rise in song. You cannot make out what it is saying, but it carries the tones of anticipation. You can, however, clearly decipher a chant from many other voices, cutting through the solo voice in time with the drumming but not overwhelming either:`

Reviewer: ```suggestion
			`	Out of the chaos you hear a voice rise in song. You cannot make out what it is saying, but it carries the tones of anticipation. You can, however, clearly decipher a chant from many other voices, cutting through the solo voice in time with the drumming:`
```
Personal preference. That last bit feels too run-onish; I liked the original flow better.

			`	Taely notices your anxiety and gestures reassuringly. "They sing of the days of humanity's infancy; when we first ventured out of the cradle."`
			label countdownannotated
			`	On the final word the chant ends and dozens of deep voices join together in a roaring crescendo before fading away, replaced by many new voices expressing tones of wonder. At this point, it seems that almost every single person in the room is drumming, dancing, or singing - many seem to be doing several simultaneously.`
			`	As the stars wheel through the night sky above the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of words, but the emotion conveyed in the singing is clear. The early tone of anticipation shifts to the roar of excitement and then on to a peaceful tone of wonder.`

> 			`	As the ceremony continues, this shifts again to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in duel tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other deeply emotional pairings before finally culminating in a return to the original, unified heartbeat as the singers' voices come together, reaching upwards in hope before finally everything trails off into silence.`

Reviewer: ```suggestion
			`	As the ceremony continues, this shifts again to a steady, almost mechanical feel, which is in turn abruptly penetrated by a piercing thread of excitement, leading to the entire tone of the song becoming one of curiosity over another driving beat. The melody then seems to wander more, caught in dual tones of regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other deeply emotional pairings before finally culminating in a return to the original, unified heartbeat as the singers' voices come together, reaching upwards in hope before finally everything trails off into silence.`
```
`duel` with an `e` is a fight between two people. `dual` with an `a` means two things.
